Sources/ResourceProvider/Cache/Samples/NSCacheCache.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  NSCacheCache.swift
+//  swift-resource-provider
+//
+//  Created by Óscar Morales Vivó on 11/14/24.
+//
+
+import Foundation
+
+/**
+ In-memory reference type cache for objects.
+
+ Yes, the name is pretty stupid.
```

Comment:
😄

Reply:
I'll take less stupid ideas if you got any 😅